src/scene/gsplat/gsplat-compressed-data.js

```diff
@@ -9,9 +9,34 @@ import { GSplatData } from './gsplat-data.js';
 
 const SH_C0 = 0.28209479177387814;
 
+// https://fgiesen.wordpress.com/2012/03/28/half-to-float-done-quic/
```

Comment:
maybe should be moved to FloatPacking module, along the `float2Half`

Reply:
done